src/Symfony/Component/Form/Util/ServerParams.php

```diff
@@ -26,6 +26,19 @@ public function __construct(RequestStack $requestStack = null)
     }
 
     /**
+     * Returns true if the POST max size has been exceeded in the request.
+     *
+     * @return bool
+     */
+    public function hasPostMaxSizeBeenExceeded()
+    {
+        $contentLength = $this->getContentLength();
+        $maxContentLength = $this->getPostMaxSize();
+
+        return !empty($maxContentLength) && $contentLength > $maxContentLength;
```

Comment:
If it can be an int or null then empty is entirely useful as it returns true for 0 and null...